pkg/capacity/capacity_test.go

```diff
@@ -96,6 +123,9 @@ func TestController(t *testing.T) {
 		modify             func(ctx context.Context, clientSet *fakeclientset.Clientset, expected []testCapacity) (modifiedExpected []testCapacity, err error)
 		capacityChange     func(ctx context.Context, storage *mockCapacity, expected []testCapacity) (modifiedExpected []testCapacity)
 		topologyChange     func(ctx context.Context, topology *topology.Mock, expected []testCapacity) (modifiedExpected []testCapacity)
+
+		expectedObjectsPrepared objects
+		expectedTotalProcessed  int64
```

Comment:
nit: name it something like `expectedTotalFinal`. "Total Processed"  could also imply the number of objects processed

Reply:
"Processed" refers to the step when the check runs (= after processing the work queue). If the tests ever get extended to do more work, the "final" might become misleading. I therefore prefer "Processed".